src/app/home-page/new-dashboard/new-dashboard.component.html

```diff
@@ -8,7 +8,7 @@
       <app-entry-box class="fx-grow box-container" entryType="TOOL"></app-entry-box>
       <app-entry-box class="fx-grow box-container" entryType="SERVICE"></app-entry-box>
```

Comment:
Could try using `fxFlex` for all three `app-entry-box` components so the `fx-grow` and `box-container` classes can be removed